Index.HTML

```diff
@@ -0,0 +1,119 @@
+            </div>
+            <div id="below-vid">
+
+                <div id="name-div">
+                    <h1 id="video-name">Contributing to Open Source Part 1: The Easy Way</h1>
+                    <p id="views"><big>1,000,000</big> views</p>
+                </div>
+                <div id=like-div>
+                    <button type="submit" id="like-button" class="borderless">
+                        <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false"
+                            xmlns="http://www.w3.org/2000/svg">
+                            <path 
+                                d="M1 21h4V9H1v12zm22-11c0-1.1-.9-2-2-2h-6.31l.95-4.57.03-.32c0-.41-.17-.79-.44-1.06L14.17 1 7.59 7.59C7.22 7.95 7 8.45 7 9v10c0 1.1.9 2 2 2h9c.83 0 1.54-.5 1.84-1.22l3.02-7.05c.09-.23.14-.47.14-.73v-1.91l-.01-.01L23 10z" 
+                                fill="rgb(70,70,70)">
+                            </path>
+                        </svg>
+                    </button>
+                    <button type="submit" id="dislike-button" class="borderless">
+                        <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false"
+                            xmlns="http://www.w3.org/2000/svg">
+                            <path
+                                d="M15 3H6c-.83 0-1.54.5-1.84 1.22l-3.02 7.05c-.09.23-.14.47-.14.73v1.91l.01.01L1 14c0 1.1.9 2 2 2h6.31l-.95 4.57-.03.32c0 .41.17.79.44 1.06L9.83 23l6.59-6.59c.36-.36.58-.86.58-1.41V5c0-1.1-.9-2-2-2zm4 0v12h4V3h-4z"
+                                fill="rgb(70,70,70)">
+                            </path>
+                        </svg>
+                    </button>
+                    <button type="submit" id="share-button" class="borderless">
+                        <svg viewBox="0 0 24 24" preserveAspectRatio="xMidYMid meet" focusable="false"
+                            xmlns="http://www.w3.org/2000/svg">
+                            <path d="M14 9V5l7 7-7 7v-4.1c-5 0-8.5 1.6-11 5.1 1-5 4-10 11-11z" fill="rgb(70,70,70)"></path>
+                        </svg>
+                    </button>
+                </div>
+
+
+            </div>
+
+
+            <div id="uploader-info">
+                <div id="upavatar-div">
+                    <img src="/Icons/Uploader avatar.PNG" id="uploader-avatar" alt="Uploader avatar">
+                </div>
+                <div id="upavatar-info-div">
+                    <img id="upavatar-info" src="Thumbnails/Uploaderinfo.png" alt="Video description">
+                </div>
+                <div id="subscribe-div">
+                    <button type="submit" id="subscribe">SUBSCRIBE</button>
+                </div>
+                
+            </div>
+        </div>
+
+        <div class="recommended-videos">
+            <div>
+                <img id="autoplay" src="Thumbnails/autoplay.png" alt="Autoplay mimick">
```

Comment:
**`[OPTIONAL]`** We can use `<aside>` for the recommended videos for more semantic tags.